.travis.yml

```diff
@@ -13,38 +13,23 @@ jobs:
         - IMAGE_REPO="linux-utils"
         - TRIVYARCH="64bit"
         - RELEASE_TAG_DOWNSTREAM=1
-    - os: linux
-      arch: arm64
-      env:
-        - IMAGE_REPO="linux-utils-arm64"
-        - TRIVYARCH="ARM64"
-        - RELEASE_TAG_DOWNSTREAM=0
-    - os: linux
-      arch: ppc64le
-      env:
-        - IMAGE_REPO="linux-utils-ppc64le"
-        - RELEASE_TAG_DOWNSTREAM=0
 
 before_install:
   - if [ -z $IMAGE_ORG ]; then
       IMAGE_ORG="openebs";
       export IMAGE_ORG;
     fi
   - export DIMAGE="${IMAGE_ORG}/${IMAGE_REPO}"
-  - if [ "$TRAVIS_CPU_ARCH" != "ppc64le" ]; then
-      export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/');
-      wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-${TRIVYARCH}.tar.gz;
-      tar zxvf trivy_${VERSION}_Linux-${TRIVYARCH}.tar.gz;
-    fi  
+  - export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
+  - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-${TRIVYARCH}.tar.gz
+  - tar zxvf trivy_${VERSION}_Linux-${TRIVYARCH}.tar.gz
+
 
 script:
   - make image
   - make test
-  - if [ "$TRAVIS_CPU_ARCH" != "ppc64le" ]; then
-      ./trivy --exit-code 0 --severity HIGH --no-progress ${DIMAGE}:ci;
-      ./trivy --exit-code 1 --severity CRITICAL --no-progress ${DIMAGE}:ci;
-    fi
-  - make push
```

Comment:
Just wondering if we should still push an amd64 image to quay.io alone. this can be achieved by removing the docker creds on travis and only having the quay creds with the current script. 

This is if the buildx image can't be pushed to quay. Is this still true? @akhilerm

Reply:
> This is if the buildx image can't be pushed to quay. Is this still true?

Buildx does not support changing the repository after building. You will have to rebuild again. So multiarch image will only be pushed docker hub.  If you want to push to quay also, it needs some third party scripts (IMO, its better we dont use that).


I will enable the push from travis also, You can keep it only for quay.

Reply:
@kmova  Have added the push script. Can you remove the docker  cred from travis, so we don't accidentally overwrite the multiarch image.